src/components/flat-table/flat-table-row/flat-table-row.spec.js

```diff
@@ -675,39 +779,12 @@ describe("FlatTableRow", () => {
 
       assertStyleMatch(
         {
-          borderLeft: "2px solid var(--colorsUtilityMajor100)",
+          borderRight: "2px solid var(--colorsUtilityMajor100)",
         },
         wrapper,
-        { modifier: `:focus ${StyledFlatTableRowHeader}` }
+        { modifier: `${StyledFlatTableRowHeader}:nth-child(4)` }
       );
     });
-
-    it.each(["dark", "light", "transparent-white", "transparent-base"])(
```

Comment:
I'm a bit puzzled why these have been removed as the changes in this PR don't remove any of this styling do they?

Reply:
The styles have changed so the assertion was no longer relevant, there's little value in adding a test back in as these styles are already caught by visual regression and there's no coverage missing either